src/main/java/Task.java

```diff
@@ -0,0 +1,33 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getDescription() {
+        return description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void markAsDone() {
+        isDone = true;
+    }
+
+    public void markAsNotDone() {
+        isDone = false;
+    }
+
+    public String getStatus() {
+        return (isDone ? "[X]" : "[ ]");
+    }
+
+    public String getType(){
+        return "0";
+    }
```

Comment:
```suggestion
    public String getType() {
        return "[0]";
    }
```
The subclass overridden methods, return string with "[]"; consider using it here as well to make code look consistent. Also, make use to add space before braces.